src/components/LogIn/index.js

```diff
@@ -62,6 +57,9 @@ const LogIn = props => {
               {errors.email &&
                 errors.email.type === 'loginError' &&
                 'Tarkista, että kirjoitit sähköpostin oikein.'}
+              {errors.email &&
```

Comment:
Verifikoida might be a little hard for our target group. Maybe something like: "Sähköpostia ei ole vielä vahvistettu. Vahvista sähköposti ensin." etc.